src/Atoms/Inputs/Password.js

```diff
@@ -43,26 +38,30 @@ class Password extends React.Component<PropsType, StateType> {
   }
 
   handleSwitchClick = () => {
-    if (this.state.type === 'password') {
-      this.setState({ type: 'text' })
-    } else {
-      this.setState({ type: 'password' })
-    }
+    this.state.type === 'password'
+      ? this.setState({ type: 'text' })
+      : this.setState({ type: 'password' })
+  }
+
+  renderToggle = (disabled?: boolean) => {
+    return disabled === false ? (
+      <Switch onClick={this.handleSwitchClick}>
+        {this.state.type === 'password' ? 'Show' : 'Hide'}
+      </Switch>
+    ) : null
   }
 
   render() {
     const { type } = this.state
-    const { disabled, width } = this.props
+    const { width } = this.props
```

Comment:
you dont need to pass width explicitly anymore width will be passed with `...this.props`